datafn/python/tests/mock_db.py

```
@@ -0,0 +1,101 @@
+        records = self.data.get(model, [])
+        filtered = []
+        for r in records:
+            match = True
+            for w in where:
+                field = w["field"]
+                op = w.get("operator", "eq")
+                val = w.get("value")
+                
+                rv = r.get(field)
+                if op == "eq":
+                    if rv != val: match = False
```

Ruff (0.15.7)

[error] Multiple statements on one line (colon)

(E701)